pkgs/applications/office/paperwork/default.nix

```diff
@@ -46,9 +54,27 @@ python3Packages.buildPythonApplication rec {
     paths = lib.collect lib.isDerivation aspellDicts;
   }}/lib/aspell";
 
+  postInstall = ''
+    # paperwork-shell needs to be re-wrapped with access to paperwork
+    cp ${python3Packages.paperwork-backend}/bin/.paperwork-shell-wrapped $out/bin/paperwork-shell
+    # install desktop files and icons
+    # paperwork-shell install copies everything to $HOME/.local/share instead of $out/share
+    mkdir -p home/.local
```

Comment:
Hm, can't this be directly patched instead?

Reply:
Patching is not better, because the user is supposed to be able to call paperwork-shell install.